app-modules/division/database/migrations/2023_09_15_201951_create_divisions_table.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+return new class () extends Migration {
+    public function up(): void
+    {
+        Schema::create('divisions', function (Blueprint $table) {
+            $table->uuid('id')->primary();
+
+            $table->string('name');
+            $table->string('code')->unique(); //unique?
+            $table->longText('header')->nullable();
+            $table->longText('footer')->nullable();
+
+            $table->foreignUuid('created_by_id')->nullable()->constrained('users');
+            $table->foreignUuid('updated_by_id')->nullable()->constrained('users');
```

Comment:
I know this is how it was defined in the ticket, but I'm wondering if this should explicitly be `last_updated_by_id`, even though it's fairly implicit from the nature of updating.